go/routers/app_multiplexer/tests/test_views.py

```diff
@@ -30,7 +30,11 @@ def test_show_stopped(self):
         router = response.context[0].get('router')
         self.assertEqual(router.name, u"myrouter")
         self.assertContains(response, rtr_helper.get_view_url('start'))
-        self.assertNotContains(response, rtr_helper.get_view_url('stop'))
+        self.assertContains(
+            response,
+            '<button class="btn btn-danger action" data-action="stop" ' +
+            ' data-url="%s" disabled>Deactivate</button>'
+            % rtr_helper.get_view_url('stop'), html=True)
```

Comment:
We should avoid assertions on things that can change for reasons unrelated to the functionality we're testing. That said, I don't really see a better way to do this. :-/